BrowserSelect/frm_settings.Designer.cs

```diff
@@ -203,53 +219,70 @@ partial class frm_settings {
             this.browser.DataPropertyName = "Browser";
             this.browser.HeaderText = "Browser";
             this.browser.Name = "browser";
-            this.browser.Width = 51;
+            this.browser.Width = 73;
             // 
             // groupBox4
             // 
             this.groupBox4.Controls.Add(this.chk_check_update);
             this.groupBox4.Controls.Add(this.btn_check_update);
-            this.groupBox4.Location = new System.Drawing.Point(12, 272);
+            this.groupBox4.Location = new System.Drawing.Point(18, 418);
+            this.groupBox4.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
             this.groupBox4.Name = "groupBox4";
-            this.groupBox4.Size = new System.Drawing.Size(150, 41);
+            this.groupBox4.Padding = new System.Windows.Forms.Padding(4, 5, 4, 5);
+            this.groupBox4.Size = new System.Drawing.Size(225, 63);
             this.groupBox4.TabIndex = 5;
             this.groupBox4.TabStop = false;
             this.groupBox4.Text = "Update checker";
             // 
+            // chk_check_update
+            // 
+            this.chk_check_update.AutoSize = true;
+            this.chk_check_update.Location = new System.Drawing.Point(18, 25);
+            this.chk_check_update.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
+            this.chk_check_update.Name = "chk_check_update";
+            this.chk_check_update.Size = new System.Drawing.Size(83, 24);
+            this.chk_check_update.TabIndex = 1;
+            this.chk_check_update.Text = "enable";
+            this.chk_check_update.UseVisualStyleBackColor = true;
+            this.chk_check_update.CheckedChanged += new System.EventHandler(this.chk_check_update_CheckedChanged);
+            // 
             // btn_check_update
             // 
-            this.btn_check_update.Location = new System.Drawing.Point(69, 12);
+            this.btn_check_update.Location = new System.Drawing.Point(104, 18);
+            this.btn_check_update.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
             this.btn_check_update.Name = "btn_check_update";
-            this.btn_check_update.Size = new System.Drawing.Size(75, 23);
+            this.btn_check_update.Size = new System.Drawing.Size(112, 35);
             this.btn_check_update.TabIndex = 0;
             this.btn_check_update.Text = "check now";
             this.btn_check_update.UseVisualStyleBackColor = true;
             this.btn_check_update.Click += new System.EventHandler(this.btn_check_update_Click);
             // 
-            // chk_check_update
+            // button2
```

Comment:
maybe rename button2 to btn_refresh?

Reply:
Sorry, kind of new to pull request management - do I make these changes in the original branch and send a new PR? Can I update the existing PR?

Reply:
https://stackoverflow.com/a/16748236/1461004